composer.json

```diff
@@ -21,10 +21,26 @@
         }
     ],
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
```

Comment:
I think we should go to 7.2

Reply:
This is just an import. Improvements in code should be done afterwards

Reply:
Ok but before the first release

Reply:
Ok but before the first release